### ISSUE TITLE
AKAuthenticationError Code=-7003 "(null)"

### ISSUE THREAD

**Issue by glavermicocca** [mirzemehdi/KMPAuth#34]
I probably forgot someting... any clue about this errore? 🤯

Authorization failed: Error Domain=AKAuthenticationError Code=-7003 "(null)" UserInfo={AKClientBundleID=com.mmk.kmpauth...

it's mandatory to create serviceId in develop.apple.com ? or it's sufficent to create appid?

for KmpNotification proble was "APNs Authentication Key" that i forgot to add in order to make secure connection between firebase and apple (if i well undestand) - but now for auth i'm working 2 day around this error that i'm not able to came out!

p.s. Google Auth work like a charm!

<img width="347" alt="image" src="https://github.com/mirzemehdi/KMPAuth/assets/10889124/ed92a4ef-ab83-4c62-b34b-fba2a4854235">

<img width="729" alt="image" src="https://github.com/mirzemehdi/KMPAuth/assets/10889124/f1ca94b4-fd25-4d7d-9842-a1cdacf6b784">

<img width="400" alt="image" src="https://github.com/mirzemehdi/KMPAuth/assets/10889124/c096c8d2-e8fb-455b-ace7-579e267f5b33">

**Comment by glavermicocca** [comment]
Adding some details about this error: I'm using an old iPhone 6s (iOS version 15.8). When I tried to log in with my Apple ID from a simulator (iOS version 17.5), it probably unlocked the situation. After this step with the updated simulator, I was able to log in from the physical iPhone 6s device as well. A very strange situation. Finally, I add one more detail about the "minimum deployment version": I had set it to 15.6 to be able to run it on the iPhone 6s, and perhaps this caused the error 7003. I will do some further investigation in the coming days to rule out other configuration errors, for secure there aren't problems related to "Certificates, Identifiers & Profiles" configurations.
